### PR TITLE
Mu 19

### PR DESCRIPTION
**Pull request by Staroz**
Check this PR, I've been fixing issues from GitHub Actions and I hope to improve this. thank you.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "cSpell.words": [
+        "cription"
+    ]
+}
```
Comment (sphinix27) [inline .vscode/settings.json]: Is this required?

File: cypress/screenshots/updateCards.cy.js/Test for update information of cards in Trello -- Add description in a card -- after all hook (failed).png
Comment (sphinix27) [inline cypress/screenshots/updateCards.cy.js/Test for update information of cards in Trello -- Add description in a card -- after all hook (failed).png:1]: Remove all these png files.

File: playwright-report/index.html
Comment (sphinix27) [inline playwright-report/index.html:1]: Better to remove the folder `playwright-report`

File: tests-examples/demo-todo-app.spec.js
Comment (sphinix27) [inline tests-examples/demo-todo-app.spec.js:1]: Remove this file

File: tests/cardsTrello.spec.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import { test, expect } from '@playwright/test';
+const  credentials = require('../cypress/fixtures/credentials1.json');
+const { chromium } = require('@playwright/test');
+const { LoginPage } = require('./pages/loginPage.page');
+const { WorkspaceApi } = require('./pages/workspaceApi');
+const { BoardsApi } = require('./pages/boardsApi');
+const { CardsUi } = require('./pages/cardsUi')
+const axios = require('axios');
```
Comment (sphinix27) [inline tests/cardsTrello.spec.js]: Is this import required?

File: tests/listsTrello.spec.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import { test, expect } from '@playwright/test';
+const  credentials = require('../cypress/fixtures/credentials1.json');
+const { chromium } = require('@playwright/test');
+const { LoginPage } = require('./pages/loginPage.page');
+const { WorkspaceApi } = require('./pages/workspaceApi');
+const { BoardsApi } = require('./pages/boardsApi');
+const { ListsUi } = require('./pages/listsUi');
+const axios = require('axios');
```
Comment (sphinix27) [inline tests/listsTrello.spec.js]: Is this import required?

File: tests/testTrello.spec.js [hunk 4]
```diff
@@ -4,41 +4,47 @@ const { chromium } = require('@playwright/test');
 const { BoardsUi } = require('./pages/boardsUi.js');
 const { BoardsApi } = require('./pages/boardsApi');
 const { LoginPage } = require('./pages/loginPage.page');
+const { WorkspaceApi } = require('./pages/workspaceApi')
 const axios = require('axios');
```
Comment (sphinix27) [inline tests/testTrello.spec.js]: Is this import required?

**Review by sphinix27 — CHANGES_REQUESTED** [review]
Apply suggested changes and fix PR issues

**Comment by Staroz** [comment]
Hi, I've fixed the issues that you saw in my PR, please, review again this PR.

File: cypress/screenshots/updateCards.cy.js/Test for update information of cards in Trello -- Add description in a card -- after all hook (failed).png
Comment (sphinix27) [inline cypress/screenshots/updateCards.cy.js/Test for update information of cards in Trello -- Add description in a card -- after all hook (failed).png:1]: Still these files are added, they should be removed.

File: cypress/videos/updateCards.cy.js.mp4
Comment (sphinix27) [inline cypress/videos/updateCards.cy.js.mp4:1]: Remove this file

File: playwright-report/index.html
Comment (sphinix27) [inline playwright-report/index.html:1]: Remove this file

File: tests-examples/demo-todo-app.spec.js
Comment (sphinix27) [inline tests-examples/demo-todo-app.spec.js:1]: Remove this file

File: tests/updateCards.spec.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
+import { test, expect } from '@playwright/test';
+const  credentials = require('../cypress/fixtures/credentials1.json');
+const { chromium } = require('@playwright/test');
+const { LoginPage } = require('./pages/loginPage.page');
+const { WorkspaceApi } = require('./pages/workspaceApi');
+const { BoardsApi } = require('./pages/boardsApi');
+const { CardsUi } = require('./pages/cardsUi')
```
Comment (sphinix27) [inline tests/updateCards.spec.js:7]: In the future when creating Page Object classes, it is better to just name them like: `CardsPage` or `NavPage`

**Review by sphinix27 — CHANGES_REQUESTED** [review]
Review how to remove existing files from git. Only that is remaining, everything else looks good.

**Comment by Staroz** [comment]
Thanks for the observations.

**Pull request merged**